crates/wac-resolver/src/lib.rs

```diff
@@ -149,3 +144,35 @@ pub fn packages<'a>(
     visitor.visit(document)?;
     Ok(keys)
 }
+
+#[derive(Debug, Error)]
+/// Error for WAC commands
+pub enum CommandError {
```

Comment:
This doesn't seem right. Why weren't error variants just added to the `Error` enum?